usage/installer/src/main/assembly/files/blueprints/seaclouds-on-byon.yaml

```diff
@@ -19,12 +19,16 @@ services:
   - serviceType: "classpath://brooklyn/entity/brooklynnode/brooklyn-node.yaml"
     id: deployer
     name: SeaClouds Deployer
-    install.version: 0.7.0-SNAPSHOT
+    install.version: 0.7.0-incubating
     managementUser: admin
     managementPassword: p4ssw0rd
     brooklynLocalPropertiesContents: |
       brooklyn.webconsole.security.users=admin
       brooklyn.webconsole.security.user.admin.password=p4ssw0rd
+    brooklyn.config:
+      brooklynnode.download.archive.subpath: brooklyn-dist-0.7.0-incubating/
+      brooklynnode.classpath:
+        - classpath://deployer-0.7.0-incubating.jar
```

Comment:
`deployer-${project.version}.jar` is added a resource to installer.jar, it allows to Brooklyn to find the deployer jar as a resource using the `classpath` notation. Then, as @mbarrientos mentioned, the deployer jar should be pointed as `classpath://deployer-0.8.0-SNAPSHOT.jar`, following the version of SeaCloudsPlatform.